Mathlib/Analysis/Analytic/IsolatedZeros.lean

```diff
@@ -240,6 +240,46 @@ lemma apply_eq_zero_of_order_toNat_ne_zero (hf : AnalyticAt 𝕜 f z₀) :
   simp [hf.order_eq_zero_iff]
   tauto
 
+/-- Helper lemma for `AnalyticAt.order_mul` -/
+lemma order_of_locallyZero_mul_analytic {f g : 𝕜 → 𝕜} (hf : AnalyticAt 𝕜 f z₀)
```

Comment:
I would expect the name more like `order_eq_top_of_order_eq_top_mul_analytic`...

Reply:
Changed the name following your suggestion.